random_menu_generator.rb

```diff
@@ -0,0 +1,55 @@
+# Random Menu Generator
+# This program takes user input and outputs a chosen number of unique random menu items
+
+# Introduce the generator and ask for input
+puts "Welcome to the Random Menu Generator.\n\n"
+puts "You get to pick a descriptor, a cooking style, and a type of food to combine into a random menu item."
+puts "The generator will make a list of unique menu items, so make sure to enter enough words in each category.\n\n"
+puts "Let's start with adjectives. Please enter a list of adjectives, separated by spaces:"
+
+# Gets input from the user and splits it on white space, then stores that in an array.
+adjectives = gets.chomp.split
```

Comment:
It accepts the return, and creates an empty array, which is not what I would want!

I could create a loop that checks for valid input (some sort of words) and requests new input until the input passes the test.